base.js

```diff
@@ -324,6 +327,7 @@ export default defineConfig(
           'newlines-between': 'always',
           alphabetize: {
             order: 'asc',
+            orderImportKind: 'asc',
```

Comment:
This enforces `import type X from 'specifier'` to be put before `import X from 'specifier'`. WebStorm does the same when the "Optimize imports" action is executed.